tests/unit/retrievers/test_text2cypher.py

```diff
@@ -250,3 +256,30 @@ def test_t2c_retriever_invalid_custom_prompt_type(
         )
 
     assert "Input should be a valid string" in str(exc_info.value)
+
+
+@pytest.mark.usefixtures("caplog")
+@patch("neo4j_genai.retrievers.base.Retriever._verify_version")
+def test_t2c_retriever_with_custom_prompt_kwargs_inputs(
+    _verify_version_mock: MagicMock,
+    driver: MagicMock,
+    llm: MagicMock,
+    neo4j_record: MagicMock,
+    caplog: pytest.LogCaptureFixture,
+) -> None:
+    prompt = "This is a custom prompt. {query_text} {examples}"
+    query = "test"
+    examples = ["example A", "example B"]
+    with caplog.at_level(logging.DEBUG):
+        retriever = Text2CypherRetriever(driver=driver, llm=llm, custom_prompt=prompt)
+        driver.execute_query.return_value = (
+            [neo4j_record],
+            None,
+            None,
+        )
+        retriever.search(query_text=query, examples=examples)
+
+    assert (
```

Comment:
Generally speaking, I'm not sure we really care about what's in the logs. However you're right we need to test that the correct prompt was used. Since we have a mocked `llm` object, we can test that it was called with the correct prompt instead, something like this:
```
llm.invoke.assert_called_once_with("This is a custom prompt. test ['example A', 'example B']")
```